test/CMakeLists.txt

```diff
@@ -81,21 +81,6 @@ if (APPLE)
     set(NGRAPH_TF_CXX11_ABI 0)
 endif()
 
-# If pre-built tensorflow CC library is not provided, then build one.
-# TODO: Remove this option once we migrate to build_ngtf.py 
-if (NOT UNIT_TEST_TF_CC_DIR)
-    add_custom_target(
-        build_tf_cc
```

Comment:
Note to self: `build_tf_cc` is being removed, hence the whole file is being deleted